pkg/acsengine/const.go

```diff
@@ -32,7 +32,7 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0-beta.1"
```

Comment:
This will have to wait for the official 1.6 stable release.  Also you will need to adjust all 1.5.* references everywhere including the dashboard, documenation, kubectl download links, windows binaries.  It may actually be best to submit as 2 PRs, one from v1.6, and one for GPU support.